### PR TITLE
Fix admin dashboard discussions and issues listing

### PR DESCRIPTION
**Pull request by noirbizarre**
This PR mutualize the current user issues and discussions listing and fix the admin dashboard discussions and issues listing.

### CODE REVIEW

File: udata/core/discussions/actions.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+def discussions_for(user, closed=False):
+    '''
+    Build a queryset to query discussions related to a given user's assets.
+
+    It includes discussions coming from the user's organizations
+
+    :param bool closed: whether to include closed discussions or not.
+    '''
+    orgs = [o for o in user.organizations if o.is_member(user)]
```
Comment (davidbgk) [inline udata/core/discussions/actions.py]: User can have attached organizations that he is not member of?

File: udata/core/discussions/actions.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals, absolute_import
+
+from udata.models import Reuse, Dataset
+
+from .models import Discussion
+
+
+def discussions_for(user, closed=False):
```
Comment (davidbgk) [inline udata/core/discussions/actions.py]: `only_open=True` is more explicit and avoid double negation.

File: udata/core/issues/actions.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals, absolute_import
+
+from udata.models import Reuse, Dataset
+
+from .models import Issue
+
+
+def issues_for(user, closed=False):
```
Comment (davidbgk) [inline udata/core/issues/actions.py]: Same here.

File: udata/core/discussions/actions.py (hunk 1 above)
Comment (noirbizarre) [inline udata/core/discussions/actions.py]: No, you're right. :)

**Comment by davidbgk** [comment]
r+wc

File: udata/core/discussions/actions.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+from .models import Discussion
+
+
+def discussions_for(user, only_open=True):
+    '''
+    Build a queryset to query discussions related to a given user's assets.
+
+    It includes discussions coming from the user's organizations
+
+    :param bool closed: whether to include closed discussions or not.
```
Comment (davidbgk) [inline udata/core/discussions/actions.py]: Update docs too ;)

File: udata/core/issues/actions.py [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+from .models import Issue
+
+
+def issues_for(user, only_open=True):
+    '''
+    Build a queryset to query issues for a given user's assets.
+
+    It includes issues coming from the user's organizations
+
+    :param bool closed: whether to include closed issues or not.
```
Comment (davidbgk) [inline udata/core/issues/actions.py]: Same here

**Pull request merged**